README.md

```diff
@@ -74,6 +74,15 @@ To contribute to diagram, check out [contribution guidelines](CONTRIBUTING.md).
 
 [Cloudiscovery](https://github.com/Cloud-Architects/cloudiscovery) helps you to analyze resources in your cloud (AWS/GCP/Azure/Alibaba/IBM) account. It allows you to create a diagram of analyzed cloud resource map based on this Diagrams library, so you can draw the your existing cloud infratructure with Cloudicovery.
 
+## Breaking Changes
```

Comment:
What do you think about creating a `CHANGELOG.md` (under root directory) for logging all changelogs for each version including the breaking changes. This is an example:

**CHANGELOG.md**

```markdown
# Changelogs

## 0.16.0

### Breaking Changes

- ...

### Added

- ...

### Fixed

- ...
```

Here is a reference of Kubernetes's changelog file format: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md

Reply:
I added data for 2 versions for 0.15 and 0.16 by copying the commit titles... 
I added an issue to add a changelog generator, I can check about or maybe somebody that already use such tool can help and be faster to make a PR.